### ISSUE TITLE
questions for runing model

### ISSUE THREAD

**Issue by ZuoxiYang** [hyades910739/LFM1b_Recommendation#1]
Hi, thanks a lot for this amazing work. But when I ran the "train_caser.py" file, it reported that:
“
...
Now loading data from lfm_1b/train25
4279it [00:00, 132512.01it/s]
...
Now loading data from lfm_1b/test25
4279it [00:00, 158449.59it/s]
Namespace(L=5, T=3, batch_size=512, l2=1e-06, learning_rate=0.001, n_iter=10, neg_samples=3, seed=1234, test_root='lfm_1b/test25', train_root='lfm_1b/train25', use_cuda=True)
Namespace(L=5, ac_conv='relu', ac_fc='relu', d=50, drop=0.5, nh=16, nv=4)
total training instances: 30853
row:4278
col:55978
d:60806
num_seqs:4279
num_items:55979
torch.Size([512, 5])
torch.Size([512, 1])
torch.Size([512, 3])
torch.Size([512, 9])”

So how to print more information about "train model", "evaluate mode" and so on?
